src/lib/components/engineUI/EngineSeperate.svelte

```diff
@@ -0,0 +1,372 @@
+<script lang="ts">
+	import Modal from "../dialogPopover/Modal.svelte";
+	import type { EngineDTO } from "./EngineDTO";
+	import {
+		Delete,
+		Close,
+		Done,
+		Check_box,
+		Check_box_outline_blank,
+	} from "svelte-google-materialdesign-icons";
+	import type iModalComponent from "$lib/interfaces/IModalComponent";
+	import Button from "../overlayMenu/elements/Button.svelte";
+	import {
+		comparePortRange,
+		validateEndPort,
+		validateIP,
+		validateName,
+		validateStartPort,
+	} from "$lib/classes/engine/Validation";
+
+	let formElement: HTMLFormElement;
+	let modalContainer: Modal & iModalComponent;
+	let nameContainer: HTMLInputElement;
+	let ipAddressContainer: HTMLInputElement;
+	let startPortContainer: HTMLInputElement;
+	let endPortContainer: HTMLInputElement;
```

Comment:
Instead of binding to HTML elements you should use the bind:value. This would simplify the code quite a lot!